### PR TITLE
Edit user

### PR DESCRIPTION
**Pull request by afmicc**
## Edit user

Trello references: 
* [As a user I should be able to edit my profile so I can update my personal information or fill incomplete fields](https://trello.com/c/93dRLIV5)
* [As a user I should be able to upload a picture to my profile](https://trello.com/c/NqIharwn)

#### Description

* Add user update test cases
* Add user picture field
* Update `requirements.txt`

### CODE REVIEW

File: applications/users/serializers.py [hunk 1]
```diff
@@ -9,7 +9,8 @@
 class UserSerializer(UserDetailsSerializer):
     class Meta:
         model = User
-        fields = ['id', 'email', 'name', 'gender']
+        fields = ['id', 'email', 'name', 'gender', 'picture', ]
+        read_only_fields = ['email', ]
```
Comment (brunomichetti) [inline applications/users/serializers.py]: `id` should be read_only too

File: applications/users/tests/test_user_update.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        prev_gender = self.user.gender
+
+        response = self.call_update()
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(self.params['email'], prev_email)
+        self.assertNotEqual(self.params['name'], prev_name)
+        self.assertNotEqual(self.params['gender'], prev_gender)
+
+    def test_missing_name_repond_success_and_not_change_data(self):
```
Comment (brunomichetti) [inline applications/users/tests/test_user_update.py]: A little typo: `repond`

File: applications/users/tests/test_user_update.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            ['id', 'email', 'name', 'gender', 'picture', ]
+        )
+
+    def test_all_params_valid_save_data(self):
+        response = self.call_update()
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(self.params['email'], self.user.email)
+        self.assertEqual(self.params['name'], self.user.name)
+        self.assertEqual(self.params['gender'], self.user.gender)
```
Comment (brunomichetti) [inline applications/users/tests/test_user_update.py]: Yo should test that the user has the changed values in the database

**Review by brunomichetti — CHANGES_REQUESTED** [review]

File: applications/users/tests/test_user_update.py (hunk 3 above)
Comment (guilleijo) [reply applications/users/tests/test_user_update.py]: As Bruno said, you should get the user from the db and check against that data.

**Review by guilleijo — APPROVED** [review]

**Review by guilleijo — APPROVED** [review]

**Review by brunomichetti — APPROVED** [review]

**Pull request merged**